lib/unique-names-generator.ts

```diff
@@ -7,19 +7,24 @@ export interface Config {
   separator: string;
   length: number;
   dictionaries: string[][];
+  random: boolean,
 }
 
 export class UniqueNamesGenerator {
   private dictionaries: string[][];
   private length: number;
   private separator: string;
-
+  private random: boolean;
+  private generateRandomNumber(low: number = 0, high: number) {
+    return Math.floor(Math.random()*(high - low)) + low;
+  }
```

Comment:
move this private method at the bottom of the class after the `generate` public one